exonum/src/api/private/system.rs

```diff
@@ -17,12 +17,11 @@ use std::collections::HashMap;
 
 use router::Router;
 use iron::prelude::*;
-use params::{Params, Value as ParamsValue};
 
 use crypto::PublicKey;
 use node::{ExternalMessage, ApiSender};
 use blockchain::{Service, Blockchain, SharedNodeState};
-use api::{Api, ApiError};
+use api::{Api, ApiError, required_param};
```

Comment:
```rust
use api::{..., self};
api::required_param(...);
```
vs
```rust
use api::{..., required_param};
required_param(...);
```
Do we have some conventions here?..

Reply:
I guess we DON'T 😄

Reply:
Should we? :smile: